jsapp/scss/_variables.scss

```diff
@@ -12,7 +12,7 @@ $drawer-background: colors.$kobo-gray-96;
 $drawer-icons-width: 58px;
 $s-drawer-width: 270px;
 
-$mdl-layout-width: 1440px;
+$mdl-layout-width: 1920px;
```

Comment:
We could probably go all the way to `100%`.

I was putting off doing this because I thought we should check / fine tune screens like account settings, billing, but if there's a problem we could do a follow-up PR.

Making this full width now would improve the experience all around, particularly for project tables.